docs/site/getting-started/getting-started-shopping-cart-app.md

```diff
@@ -244,7 +244,7 @@ Like a regular microfrontend, an activator must connect to the platform host to
 
 In this section, we will listen for messages published to the topic `shopping-cart/add-product` and put the message's payload into session storage. Like in the previous chapter, we use the `ShoppingCartService`.
```

Comment:
There seems to be a line-ending issue as well, because the last dot is marked red.

Reply:
I checked the line endings, they are all "LF" just as we want them. Strange that we have a change here.